src/main/java/net/boomerangplatform/mongo/model/QuotasResponse.java

```diff
@@ -0,0 +1,15 @@
+package net.boomerangplatform.mongo.model;
```

Comment:
Move QuotasResponse from net.boomerangplatform.mongo.model. Only mongo models should be in this package